sdk/schemaregistry/azure-schemaregistry-avroserializer/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.0.0b5 (Unreleased)
```

Comment:
@rakshith91 - qq: do you know what the version number here would be since we're still doing beta releases? I'm guessing 1.1.0b1?

Reply:
this should only increase beta version, so this should be right.